migrations/001_documents.up.sql

```diff
@@ -0,0 +1,9 @@
+create table documents(
+  document_id varchar(40) primary key not null,
```

Comment:
@mfdeveloper508 We need to integrate this with the endpoint that saves PDF's/JSON to the database. The calling code will store this `documentID` for lookups later on.

For example, Moov's stack will store (customerID, documentID) to retrieve all documents for a customerID.

Reply:
can we add metadata  to documents table? 
it seems that we can support lookup about meta data easily.

Reply:
What are you thinking that would look like on the table?

Reply:
I think that we should use json string for this

Reply:
Sure, but there may be multiple values. It's not efficient to store that JSON blob as a string. 

For now it's easier to rely on an external system to handle metadata for each file. Let's get them encrypted first.